<launch>

	<node name="turtlesim" pkg="turtlesim" type="turtlesim_node"/>

    	<node name="turtle_teleop_joy" pkg="turtlesim" type="turtle_teleop_key"/>

    	<node name="turtle1" pkg="mas_framework" type="spawn_turtle.py">

        	<param name="number" value="1"/>

    	</node>

	<node name="turtle2" pkg="mas_framework" type="spawn_turtle.py">

        	<param name="number" value="2"/>
		<param name="x_coordinate" value="4"/>
 		<param name="y_coordinate" value="3"/>
 		<param name="theta_coordinate" value="2"/>

    	</node>

</launch>
